modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module ClaimsApi
+  class PoaFormBuilderJob
```

Comment:
When does this job actually get triggered? Let's meet today to discuss that but my thinking is we it in the post portion of the controller when a signature block is present as we discussed earlier though we should make the md5 include the payload like we do with 526 like we mentioned yesterday.